orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -177,10 +176,14 @@ def forward(self):
         pass
 
     def _build_graph(self, config):
-        if self._runtime_options.use_static_shape:
-            self._graph_builder.build(config, self._input_info.shape)
-        else:
-            self._graph_builder.build(config)
+        print_on_exit = _logger.create_log_filter(
```

Comment:
It is not clear that print_on_exit is a callable. Shall we call it `print_on_exit_func`?